aspnetcore/client-side/yeoman.md

```diff
@@ -14,21 +14,23 @@ uid: client-side/yeoman
 ---
 # Building projects with Yeoman
 
-By [Steve Smith](http://ardalis.com), [Scott Addie](https://scottaddie.com), [Rick Anderson](https://twitter.com/RickAndMSFT), [Noel Rice](http://blog.falafel.com/author/noel-rice/), and [Shayne Boyer](https://twitter.com/spboyer)
-
-[Yeoman](http://yeoman.io/) generates complete projects for a given set of client tools. Yeoman is an open-source tool that works like a Visual Studio project template. The Yeoman command-line tool [yo](https://github.com/yeoman/yo) works alongside a Yeoman generator. Generators define the technologies that go into a project.
+[Yeoman](http://yeoman.io/) is a project scaffolding system for creating many kinds of applications. Similar to Visual Studio, the Yeoman generator for ASP.NET Core contains a variety of project templates for starting a new web, MVC, or console application.
 
 ## Install Node.js, npm, and Yeoman
 
-To get started with Yeoman install [Node.js](https://nodejs.org/en/). The installer includes [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/).
+### Prerequisites
+
+Node.js and npm are required for Yeoman. Download from [Node.js](https://nodejs.org/en/), the installer includes [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/).
```

Comment:
Made the change.  Didn't like the original sentence was trying to make it read better.